Doc/library/asyncio-stream.rst

```diff
@@ -42,6 +39,19 @@ The following top-level asyncio functions can be used to create
 and work with streams:
 
 
+.. coroutinefunction:: connect(host=None, port=None, \*, \
+                               loop=None, limit=2**16, ssl=None, family=0, \
+                               proto=0, flags=0, sock=None, local_addr=None, \
+                               server_hostname=None, ssl_handshake_timeout=None, \
+                               happy_eyeballs_delay=None, interleave=None)
+
+   This function internally uses :meth:`loop.create_connection` to return an awaitable
+   :class:`Stream` object of the mode ``READWRITE`` that can be used as a reader
+   and a writer. The function can also be used as an async context manager where it's
```

Comment:
Just `The function can also be used as an async context manager`, 
drop `where it's automatically awaited`.
Technically it's not correct: `connect` returns an internal undocumented object that can be awaited or used by `async with`. I don't want to document this object but want to provide a clear vision of how to use `connect` call.

Maybe two short usage examples can help?
E.g.
```
Connect to TCP socket on *host* : *port* address, return a read-write :class:`Stream` object.

The function can be used with ``await`` to get a connected stream::
    stream = await asyncio.connect('127.0.0.1', 8888)

Also, ``async with`` form is supported, the stream is closed on exit from the block::

   async with asyncio.connect('127.0.0.1', 8888) as stream:
        ...

   *limit* determines the buffer size limit used by the returned :class:`Stream` instance. By default the limit is set to 64 KiB.

    The rest of the arguments are passed directly to :meth:`loop.create_connection`.


```

I'm not sure about the exact text but I hope you've got my idea. Documentation is hard.

Reply:
Agreed it's hard :) I have adopted your suggestion and formatted it. Did the same for `connect_unix` too.